source/upgrade/notify-admin.rst

```diff
@@ -0,0 +1,25 @@
+Notify admin
+============
+
+In some cases there are Mattermost features that are limited to specific plans. When using Mattermost, users who come across these unavailable features can send a notification to System Admins indicating that they'd like access to this feature.
+
+If access to a feature requires a plan upgrade, System Admins receive notifications of these requests in order to collect data before upgrading. For example, if only one end-user requests access to AD/LDAP, it probably isn't necessary to upgrade. However, if 20 users, including Team and Channel Admins request it, you may want to consider upgrading to support this need.
+
+This feature is designed for informational purposes only, and no action is required unless you want to take action. Your Mattermost instance's functionality is not affected if you choose not to upgrade.
+
+Notifications
+-------------
+
+Notifications are triggered by users. The very first user trigger results in a bot message to all System Admins, indicating the feature or functionality the user has requested. This bot message is listed in the direct message section of the channel sidebar.
```

Comment:
```suggestion
Notifications are triggered by users. The very first user triggers results in a bot message to all System Admins, indicating the feature or functionality the user has requested. This bot message is listed in the **Direct Messages** section of the channel sidebar.
```

Reply:
This sentence is confusing now that I re-read it. I propose:

"The very first time a user sends a request to upgrade or start a trial, a bot message is sent to all System Admins indicating the feature or functionality the user has requested that requires an upgrade or trial.

Reply:
Thanks! I agree that what's there already is clearer than my attempt to clarify ;)